packages/node-id/src/createNodeIdPlugin.ts

```diff
@@ -55,15 +52,13 @@ export const withNodeId = ({
   filter = () => true,
   allow,
   exclude,
-}: WithNodeIdProps = {}): WithOverride<HistoryEditor, NodeIdEditor> => (e) => {
-  const editor = e as typeof e & NodeIdEditor;
+}: WithNodeIdProps = {}): WithOverride<HistoryEditor> => (e) => {
+  const editor = e as typeof e & HistoryEditor;
```

Comment:
`HistoryEditor` is already in the type of e
```suggestion
  const editor = e;
```